setup.cfg

```diff
@@ -81,6 +81,36 @@ setup_requires =
     setuptools >= 38.4
     setuptools_scm
 
+[options.extras_require]
+io =
```

Comment:
`netCDF4` seems to provide wheels (certainly for linux), so it's easy to install - should we include it?

Reply:
OK will do.